src/otx/core/utils/build.py

```diff
@@ -40,6 +41,17 @@ def build_mm_model(config: DictConfig, model_registry: Registry, load_from: str
     return model
 
 
+def get_default_async_reqs_num() -> int:
```

Comment:
How about 
```suggestion
def get_default_num_async_infer_requests() -> int:
```
or `get_default_num_async_infer_reqs`?

a bit longer but easier to read

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2769/commits/0c846b7d0e0e44feaf1f1e42bfd5202b122102a3)